api/src/main/java/ca/bc/gov/educ/api/edx/orchestrator/CreateSchoolOrchestrator.java

```diff
@@ -96,20 +98,40 @@ public void createSchool(Event event, SagaEntity saga, CreateSchoolSagaData saga
 
     School school = sagaData.getSchool();
 
-    final Event.EventBuilder eventBuilder = Event.builder()
+    final Event instituteEvent = Event.builder()
       .eventType(CREATE_SCHOOL)
       .replyTo(this.getTopicToSubscribe())
       .eventPayload(JsonUtil.getJsonStringFromObject(school))
+      .sagaId(event.getSagaId())
+      .build();
+
+    this.postMessageToTopic(INSTITUTE_API_TOPIC.toString(), instituteEvent);
+    log.info("message sent to INSTITUTE_API_TOPIC for CREATE_SCHOOL Event. :: {}", saga.getSagaId());
+  }
+
+  public void checkForInitialUser(Event event, SagaEntity saga, CreateSchoolSagaData sagaData)
+  throws JsonProcessingException {
+    final SagaEventStatesEntity eventStates =
+      this.createEventState(saga, event.getEventType(), event.getEventOutcome(), event.getEventPayload());
+    saga.setSagaState(ONBOARD_INITIAL_USER.toString());
+    this.getSagaService().updateAttachedSagaWithEvents(saga, eventStates);
+
+    final Event.EventBuilder nextEventBuilder = Event.builder()
+      .eventType(ONBOARD_INITIAL_USER)
+      .replyTo(this.getTopicToSubscribe())
+      .eventPayload(JsonUtil.getJsonStringFromObject(sagaData))
       .sagaId(event.getSagaId());
 
-    if (sagaData.getInitialEdxUser().isEmpty()) {
-      eventBuilder.eventOutcome(CREATE_SCHOOL_SAGA_HAS_NO_ADMIN);
+    if (sagaData.getInitialEdxUser().isPresent()) {
+      nextEventBuilder.eventOutcome(INITIAL_USER_FOUND);
     } else {
-      eventBuilder.eventOutcome(CREATE_SCHOOL_SAGA_HAS_ADMIN);
+      nextEventBuilder.eventOutcome(NO_INITIAL_USER_FOUND);
     }
 
-    this.postMessageToTopic(INSTITUTE_API_TOPIC.toString(), eventBuilder.build());
-    log.info("message sent to INSTITUTE_API_TOPIC for CREATE_SCHOOL Event. :: {}", saga.getSagaId());
+    Event nextEvent = nextEventBuilder.build();
+    this.postMessageToTopic(this.getTopicToSubscribe(), nextEvent);
+    publishToJetStream(nextEvent, saga);
```

Comment:
Hmmm - I don't believe this is required

Reply:
@arcshiftsolutions https://github.com/bcgov/EDUC-EDX-API/pull/249/commits/198c7151d8e05a58a239c0349ebbcfcd138a71b3